### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by Jake-Shadle**
- Update scroll to 0.11
- Update clap to 3.1
- Update fuzz lockfile

This updates scroll to 0.11, which aligns with the version used by goblin (though not pdb), and clap to 3.1, only making changes to remove deprecation warnings/renames, but otherwise not changing it to the derive macros that are now built in to clap. During this I noticed that the fuzz lockfile was severely outdated so update it.

### CODE REVIEW

**Review by jan-auer — APPROVED** [review]
Thanks!

**Comment by Gankra** [comment]
Oh nice, it looks like clap 3 fixed the newlines in `--help`, so we should be able to remove my grody `\n\n\n`'s from all of the help strings. Otherwise the new output looks better.

Please also take a copy of the --help-markdown output and update the end of minidump-stackwalk/README.md with it.

**Comment by Jake-Shadle** [comment]
Oops, sorry about that, we have CLI snapshot testing as well but I'm used to OS projects not having it so didn't even bother looking, fixed now.

File: minidump-stackwalk/src/main.rs [hunk 1]
```diff
@@ -16,79 +16,80 @@ use minidump_processor::{
...
-                .long_help("Specify at a high-level how much analysis to perform.
+                .long_help(
+                    "Specify at a high-level how much analysis to perform.
 
 This flag provides a way to more blindly opt into Extra Analysis without having to know about \
 the specific features of minidump-stackwalk. This is equivalent to ProcessorOptions in \
 minidump-processor. The current supported values are:
 
-* stable-basic (default): give me solid detailed analysis that most people would want.
+* stable-basic (default): give me solid detailed analysis that most people would want
```
Comment (Gankra) [inline minidump-stackwalk/src/main.rs:92]:
```suggestion
* stable-basic (default): give me solid detailed analysis that most people would want.
```

File: minidump-stackwalk/tests/snapshots/test_minidump_stackwalk__long-help.snap [hunk 2]
```diff
@@ -1,92 +1,81 @@
...
             human usage or for checking "what's new".
             
-            Features under unstable-all may be deprecated and become noops. Features which require additional input
-            (such as `--evil-json`) cannot be affected by this, and must still be manually 'discovered'.
-            
-            
-             [default: stable-basic]  [possible values: stable-basic, stable-all, unstable-all]
+            Features under unstable-all may be deprecated and become noops. Features which require
+            additional input (such as `--evil-json`) cannot be affected by this, and must still be
+            manually 'discovered'.[default: stable-basic]
```
Comment (Gankra) [inline minidump-stackwalk/tests/snapshots/test_minidump_stackwalk__long-help.snap:76]: Eugh, is that really how clap concats the default value now?

**Comment by Gankra** [comment]
This is good enough, not gonna block on minor nits.

**Pull request merged**